docs/advanced/settings.md

```diff
@@ -503,6 +503,61 @@ This setting can only be used if the `upgrade-checker-enabled` setting is set to
 https://your.upgrade.checker-url/v99/checkupgrade
 ```
 
+### `upgrade-config`
+
+**Definition**: Upgrade-related configuration.
+
+**Default value**: `{"imagePreloadOption":{"strategy":{"type":"sequential"}}}`
+
+**Example**:
+
+```json
+{
+  "imagePreloadOption": {
+    "strategy": {
+      "type": "parallel",
+      "concurrency": 2
+    }
+  }
+}
+```
+
+#### `imagePreloadOption`
+
+**Definition**: Options for the image-preloading phase.
+
+##### `strategy`
+
+**Definition**: Image preload strategy.
+
+###### `type`
+
+**Definition**: Type of the image preload strategy.
+
+There are three available types:
+
+- `skip`: Nodes will not preload images
```

Comment:
Nodes will not preload images
->
Harvester will not preload the contaimer images from the target ISO to each node.